tuning/automation/rocblas-benchInputCreator.py

```diff
@@ -215,38 +231,140 @@ def createYaml(args, outputfile, problem, sizeMappings, verify):
 
         sizeParams = getSizeParams(size, transA, transB)
 
+        latency = 2*sizeParams['M']*sizeParams['N']*sizeParams['K']/perf[1]/1000 # us
```

Comment:
Don't we need to consider batch_count?

Reply:
I'll review the batched GEMM and will update the equation if required.

Reply:
batch count added.